.build/BuildHelper/Build-FinOpsModule.ps1

```diff
@@ -79,4 +97,7 @@ function Build-FinOpsModule
     New-ModuleManifest @manifestProperties
     Copy-Item -Path $modulePath -Destination $releasePath
     Copy-Item -Path $stringsPath -Destination $releasePath -Container -Recurse
+    Copy-Item -Path $privatePath -Destination $releasePath -Container -Recurse
+    Copy-Item -Path $publicPath -Destination $releasePath -Container -Recurse
+    Add-CopyrightHeader -Path $manifestPath
```

Comment:
This applies to everything, not just PowerShell. Shouldn't it be in a more generalized place that applies to everything?